domains/windowsed.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "windowsed1225",
+           "email": "",
+           "discord": "906415095473655810",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Efg0a1LhVERwlpp5SUaWr5ukKz0MizlImwqjnwtiHc8JCFpwy-U-b3nHiBHO1rcoT43tX5mpoBhBLl9BLNYVla6CRgtJngqAS1bajQaKP-oLRFwWp8EOLySOSR0Nm3yT82jl1ULJ-cPQF1mHtouGs3h6p0WIRtrNMc3FTumTGQft7e6tRqt5cgCjh9BukSb1wWCuBhqOz8EgS3s4zQ7YSamGHXrvX2qozbtJ5RX6wzOzq0EaKurOlZTqXY1l1o477ugTfiuZWidV0EwtT9MW40iHQmAhqLxF7MyK19fcihnzs7_Jk2rM-x_xQHLDtf69_vLxv9FAhVKPu1c1r0KOYg.Egzhvpxnuw5q5QCz3eaa-w.uWoVGC8o5fWpHxRKAc-Lez_OWuq5bQOVTzP7F1jmSgZpljr6ra2W_kyEgF2cl6bAaUMSHvo8-MqBN0g2koeG3kEyUZfpHBVrldcquIEQRHAzl1ZspSdRp8m3iJu5EwdMTxegTYeQ6k6zsW-RUNVTlg.ZxqJ8HZ6DoWHVW7goowohg"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=windowsed.is-a.dev,e5344d1f26e8286585de"
```

Comment:
Please replace the TXT record with either one of these from our Vercel docs: https://www.is-a.dev/docs/vercel/

Reply:
![image](https://github.com/user-attachments/assets/3550158e-d39c-4496-9236-e05be715eb40)
huh

Reply:
....
Thats only to **verify** your domain, which you already have `_vercel.windowsed.json`. Now you just need to replace the records with either a CNAME record of your website or a A record (but please give a preview of it)